### DISCUSSION TITLE
❌ Rendre le site Démo identique au site Prod (pages Blog & Nos Cantines)

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1152] [Idées]
Rendre le site Démo identique au site Prod pour la page Blog et, si c'est aussi possible, la page Nos Cantines pour éviter d'avoir à jongler entre les deux sites lors des démos aux utilisateurs.

**Comment by alemangui** [comment]
Identique dans quel sens ? Données ? (Car niveau fonctionnalité c'est le cas aujourd'hui)

**Reply by ghost** [reply]
Oui, identique pour les données :
- les mêmes articles sur le blog Démo que sur le blog Prod
- les mêmes cantines que sur le site Prod + toutes les cantines tests sur la page Nos Cantines de Démo

**Reply by ghost** [reply]
Pour en avoir parlé avec Jenn ce matin, elle m'a dit que faire cela serait difficile pour vous du point de vue de la gestion des 2 bases de données. Or ce n'est qu'un point de détail (cela éviterait juste d'alterner entre le site Démo et le site Prod lors des démos). Donc on oublie!
